### PR TITLE
feat: color palette v2 and grey color updates

### PR DESCRIPTION
**Pull request by shermainelim**
https://www.notion.so/701364d4fbbf4ec1801b766c589973b9?v=24aa9675206d4da9a171ca4073d81b5b&p=aab35e10a0d84dffb1fbb9f29ed3ba5b

This PR adds... 

- [x] I've kept this PR as small as possible (~600 lines) by splitting it into PRs with manageable chunks of code
- [x] I've requested reviews from 1-2 reviewers
- [ ] I've added [accessibility IDs](https://www.notion.so/dc5f4ce910f7431c84344ac79344e9f5?v=d487366816834dd4ab8dc12e0b5928f3) to my components
- [ ] I've added/updated unit/integration tests
- [ ] I've added/updated [e2e tests](https://www.notion.so/e2e-Documentation-a096d3e0bf75485e85ad692af8371ef3) for at least the happy flow
- [ ] I've added [Chinese translations for i18n setup](https://www.notion.so/Translations-Dev-Guide-8c1da838982a4f59a386ec96ac6780c8)
- [ ] I've added documentation in README/[Notion](https://www.notion.so/82b92fb1007640328dab9582c0a3694e?v=3b6ce48202cf40ad8450553799b13146)

### CODE REVIEW

File: src/common/styles/colors.tsx [hunk 1]
```diff
@@ -56,6 +57,57 @@ const palette: { [color in ColorName]: { [tone: string]: string } } = {
   },
 };
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const palettev2: { [color in ColorName]: { [tone: string]: string } } = {
+  grey: {
```
Comment (ilhaoni) [inline src/common/styles/colors.tsx:62]: hmm, i think `grey 0` is missing here

File: src/common/styles/colors.tsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ export type ColorName =
   | "green"
   | "red";
 export type ToneLevel = 0 | 10 | 20 | 30 | 40 | 50 | 60 | 80 | 100;
+export type ToneLevelv2 = 0 | 10 | 20 | 30 | 40 | 50 | 60 | 70 | 80 | 90;
```
Comment (ilhaoni) [inline src/common/styles/colors.tsx:10]: casing wise should be `ToneLevelV2` (: Do update the case for the rest of the new variables and functions too.

**Review by ilhaoni — COMMENTED** [review]
Reviewed, some small changes left 😄

File: src/components/Layout/TopBackground.tsx [hunk 3]
```diff
@@ -30,6 +30,7 @@ export const TopBackground: FunctionComponent<TopBackground> = ({
         style,
       ]}
       colors={[primaryColor, secondaryColor]}
+      //colorsv2={["#303D67", secondaryColor]}
```
Comment (ilhaoni) [inline src/components/Layout/TopBackground.tsx:33]: haha small nit here, maybe let's remove this change here? We can always add the change later on 😊 

So only `colors.tsx` will have changes for this PR (:

**Review by ilhaoni — COMMENTED** [review]
Thanks for making the changes! Just a small nit, and visual checks, and this PR should be all good (:

File: src/components/Layout/TopBackground.tsx (hunk 3 above)
Comment (shermainelim) [reply src/components/Layout/TopBackground.tsx]: have removed.

**Review by ilhaoni — APPROVED** [review]
LGTM! ^^

**Pull request merged**